scenarioo-client/app/build/use-cases-overview/use-cases-overview.component.ts

```diff
@@ -182,10 +177,6 @@ export class UseCasesOverviewComponent {
         }
     }
 
-    collapsePanel(isPanelCollapsed: boolean) {
-        this.isPanelCollapsed = isPanelCollapsed;
-    }
-
```

Comment:
cool that we got rid of all this extra code on every page :+1: 

I like in what direction this is all going :-)